src/main/menus/app.js

```diff
@@ -136,6 +136,13 @@ var createTemplate = function(mainWindow, config) {
       click: () => {
         mainWindow.webContents.send('zoom-in', -1);
       }
+    }, {
+      label: 'Zoom Out (hidden)',
+      accelerator: 'CmdOrCtrl+_',
```

Comment:
Good point -- @yuya-oc how have we handled alternate shortcuts for zoom in/out previously?